rust/connlib/clients/shared/src/messages.rs

```diff
@@ -73,24 +70,26 @@ pub struct GatewayIceCandidates {
 }
 
 /// The replies that can arrive from the channel by a client
-#[derive(Debug, Deserialize, Clone, PartialEq)]
+#[derive(Debug, Deserialize, Clone)]
 #[serde(untagged)]
 pub enum ReplyMessages {
-    ConnectionDetails(ConnectionDetails),
-    Connect(Connect),
+    UpsertFlowResult(UpsertFlowOk),
+    SetupDnsResourceNatResult(Result<(), ()>), // TODO: Ok & Err
 }
 
 // These messages can be sent from a client to a control pane
 #[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
 #[serde(rename_all = "snake_case", tag = "event", content = "payload")]
 // enum_variant_names: These are the names in the portal!
 pub enum EgressMessages {
-    PrepareConnection {
+    UpsertFlow {
```

Comment:
"Upsert" is actually wrong because we don't ever call this if we already have a flow.